### DISCUSSION TITLE
load font locally

### DISCUSSION THREAD

**Discussion by GVALFER** [imzbf/md-editor-rt#157] [General]
Hi

I do not want load the fonts remotly, can i do it locally?

Editor.mjs:45 Refused to load the script 'https://at.alicdn.com/t/c/font_2605852_prouiefeic.js' because it violates the following Content Security Policy directive: "script-src 'self' 'unsafe-eval' 'unsafe-inline' ". Note that 'script-src-elem' was not explicitly set, so 'script-src' is used as a fallback.

I want use my fonts locally, how i do that and avoid third party fonts?

**Comment by imzbf** [comment]
This example might help you:https://imzbf.github.io/md-editor-rt/en-US/docs#%F0%9F%A4%9E%F0%9F%8F%BC%20noIconfont
